mapper/orm/src/main/java/org/hibernate/search/mapper/orm/massindexing/error/ErrorHandler.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.orm.massindexing.error;
+
+/**
+ * Implementations handle errors and exceptions occurring during indexing.
+ *
+ * @author Amin Mohammed-Coleman
+ * @since 3.2
+ */
+public interface ErrorHandler {
```

Comment:
Again, this is out of scope. We even explicitly discussed it, and I told you we have a separate ticket to add back support for error handling. Please remove this commit from this PR. You can keep it somewhere for when we address error handlers, but right now it's not necessary.

Also, you took the liberty to move the error handler to the mass-indexing package. It's not specific to mass indexing at all; in fact, in Search 5 it's used for about every background work.

Reply:
OK. I ll preserve this commit too. I think we can refactor the name package when we work on the relative ticket.